packages/empty-state/src/EmptyState.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+import Image, { thumbnailSizes } from "@crave/farmblocks-image";
+import Text, { fontTypes, fontSizes } from "@crave/farmblocks-text";
+import Button, { buttonTypes, buttonSizes } from "@crave/farmblocks-button";
+
+import Container from "./styledComponents/Container";
+
+type Props = {
+  title: string,
+  width?: number,
+  imageSrc?: string,
+  description?: string,
+  primaryButtonLabel?: string,
+  onPrimaryButtonClick?: (SyntheticEvent<HTMLButtonElement>) => void,
+  info?: string
+};
+
+class EmptyState extends React.Component<Props> {
+  static defaultProps = {
+    width: 560
+  };
+
+  render() {
+    const { primaryButtonLabel, onPrimaryButtonClick } = this.props;
+    const hasPrimaryButton = primaryButtonLabel && onPrimaryButtonClick;
+    return (
+      <Container width={this.props.width}>
+        {this.props.imageSrc && (
+          <Image
+            className="thumbnail"
+            size={thumbnailSizes.LARGE}
+            src={this.props.imageSrc}
+          />
+        )}
+        {this.props.title && (
+          <Text
+            title
+            className=""
+            align="center"
+            size={fontSizes.HUGE}
+            type={fontTypes.NORMAL}
```

Comment:
Minor: `Text` component has `NORMAL` type by default... So we could remove this type unless the ideia is to make it explicity (:

Reply:
my idea was to not rely on a default value, but if we police ourselves to flag a future change in default values as breaking change, then we could review this when upgrading a major version of a dependency...